src/storage/store/var_list_column.cpp

```diff
@@ -246,6 +246,11 @@ void VarListColumn::scanFiltered(Transaction* transaction, node_group_idx_t node
         listOffset += resultVector->getValue<list_entry_t>(pos).size;
     }
 }
```

Comment:
Add an empty line here.

Reply:
Hmm. apparently clang-format >=14 can enforce this: https://clang.llvm.org/docs/ClangFormatStyleOptions.html#separatedefinitionblocks. We should probably look at upgrading at some point.